src/components/table/footer/style.css

```diff
@@ -0,0 +1,12 @@
+:host {
+	display: table-footer-group;
+	cursor: default;
+	position: sticky;
+	bottom: 0;
+	background: white ;
+	box-shadow: 0 1px 0 0 rgb(var(--smoothly-light-shade)),
+	inset  0 1px 0 rgb(var(--smoothly-light-shade));
```

Comment:
consider changing the variable to `--smoothly-color-shade` or `--smoothly-color-tint` so that it is affected by the color attribute of the table.